src/solution.c

```diff
@@ -490,23 +492,23 @@ static void solution_thread(void *arg)
     dops_t dops;
     /* Calculate the SPP position
      * disable_raim controlled by external setting. Defaults to false. */
-    s8 ret = calc_PVT(n_ready_tdcp, nav_meas_tdcp, disable_raim,
-                      &position_solution, &dops);
-    if (ret < 0) {
+    s8 pvt_ret = calc_PVT(n_ready_tdcp, nav_meas_tdcp, disable_raim,
+                          &position_solution, &dops);
+    if (pvt_ret < 0) {
       /* An error occurred with calc_PVT! */
       /* TODO: Make this based on time since last error instead of a simple
        * count. */
       /* pvt_err_msg defined in libswiftnav/pvt.c */
       DO_EVERY((u32)soln_freq,
-        log_warn("PVT solver: %s (code %d)", pvt_err_msg[-ret-1], ret);
+        log_warn("PVT solver: %s (code %d)", pvt_err_msg[-pvt_ret-1], pvt_ret);
```

Comment:
At some stage it would be nice to have a similar error message lookup for the dgnss_* functions instead of returning numbers.